ranger-plugin/service/src/main/java/org/apache/hawq/ranger/authorization/RangerHawqPluginResource.java

```diff
@@ -45,8 +47,43 @@
     /**
      * Constructor. Creates a new instance of the resource that uses <code>RangerHawqAuthorizer</code>.
      */
-    public RangerHawqPluginResource() {
+    public RangerHawqPluginResource()
+    {
+        // set UserGroupInformation under kerberos authentication
+        if (Utils.getAuth().equals("kerberos"))
```

Comment:
But I forgot to deal with lower/upper case, I will fix it and add a enum for auth method.
Thanks xiang and wen~